SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -98,6 +99,27 @@ public static partial class CsmManager
             return resourceGroup;
         }
 
+        public static async Task<ResourceGroup> LoadJenkinsResources(this ResourceGroup resourceGroup,
+            CsmWrapper<CsmJenkinsResource> jenkinsResources = null, bool load = false)
+        {
+            if (load)
+            {
+                if (jenkinsResources == null)
+                {
+                    var csmjenkinsResourcesResponse =
+                        await csmClient.HttpInvoke(HttpMethod.Get, ArmUriTemplates.JenkinsResource.Bind(resourceGroup));
```

Comment:
Yes. But to unblock them tentatively, I am using our subscription. 
I need to merge this for now for TAS deployment to PROD .